### PR TITLE
Add labels column to gcp_kms_key and gcp_compute_instance_template table. Closes #299

### LINKED ISSUE #299: Add 'labels` column to `gcp_kms_key` and `gcp_kubernetes_cluster` table.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Example query results
<details>
  <summary>Results</summary>
  
```
> select resource_labels,tags from gcp_kubernetes_cluster
+-----------------------+-----------------------+
| resource_labels       | tags                  |
+-----------------------+-----------------------+
| {"test":"test-value"} | {"test":"test-value"} |
+-----------------------+-----------------------+

> select name,labels,tags from gcp_kms_key
+------------------------------------+-------------------------------------------------------------------------------------+--------------------------------------------------------------------------------
| name                               | labels                                                                              | tags                                                                           
+------------------------------------+-------------------------------------------------------------------------------------
| turbottest6138                     | {"name":"turbottest6138"}                                                           | {"name":"turbottest6138"}                                                      
| turbottest7719                     | {"name":"turbottest7719"}                                                           | {"name":"turbottest7719"}                                                      
| turbottest8223                     | {"name":"turbottest8223"}                                                           | {"name":"turbottest8223"}                                                      
| turbottest3642                     | {"name":"turbottest3642"}                                                           | {"name":"turbottest3642"}    

> select name,labels from gcp_compute_instance_template
+-------------------------------------+----------------------+
| name                                | labels               |
+-------------------------------------+----------------------+
| gke-cluster-1-default-pool-08d80842 | {"goog-gke-node":""} |
+-------------------------------------+----------------------+




```
</details>

### CODE REVIEW

File: gcp/table_gcp_kms_key.go [hunk 1]
```diff
@@ -74,6 +74,11 @@ func tableGcpKmsKey(ctx context.Context) *plugin.Table {
 				Hydrate:     getKeyIamPolicy,
 				Transform:   transform.FromValue(),
 			},
+			{
+				Name:        "labels",
+				Description: "Labels that apply to this kms key.",
```
Comment (cbruno10) [inline gcp/table_gcp_kms_key.go:79]: @bigdatasourav Where is this description from? Is it from the SDK, API, or somewhere else?

File: gcp/table_gcp_compute_instance_template.go [hunk 2]
```diff
@@ -156,6 +156,12 @@ func tableGcpComputeInstanceTemplate(ctx context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("Properties.Tags"),
 			},
+			{
+				Name:        "labels",
+				Description: "Labels associated with the instance template.",
```
Comment (cbruno10) [inline gcp/table_gcp_compute_instance_template.go:161]: How are labels used with instance templates? From the API I can see the description:
`Labels to apply to instances that are created from these properties.`

So are these labels on the instance template themselves and/or labels applied to instances created from these templates?

Also, description wise, similar to the comment below, where did you get this comment from? The API has a different description (https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates).

File: gcp/table_gcp_kubernetes_node_pool.go [hunk 3]
```diff
@@ -32,7 +32,7 @@ func tableGcpKubernetesNodePool(ctx context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 			},
 			{
-				Name:        "selfLink",
+				Name:        "self_link",
```
Comment (cbruno10) [inline gcp/table_gcp_kubernetes_node_pool.go:35]: This should be in a different PR since it's a bug fix

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments

File: gcp/table_gcp_kubernetes_node_pool.go (hunk 3 above)
Comment (bigdatasourav) [reply gcp/table_gcp_kubernetes_node_pool.go]: will fix it in different PR.

**Review by cbruno10 — APPROVED** [review]
